Starcounter.Linq.Tests/AggregatesTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Linq;
+using Xunit;
+using static Starcounter.Linq.Tests.Utils;
+using static Starcounter.Linq.DbLinq;
+
+namespace Starcounter.Linq.Tests
+{
+    public class AggregatesTests
+    {
+        [Fact]
+        public void Count()
+        {
+            Assert.Equal("SELECT COUNT(*) FROM Starcounter.Linq.Tests.Person P",
+                Sql<Person, int>(() => Objects<Person>().Count()));
+        }
+
+        [Fact]
+        public void CountFiltering()
+        {
+            Assert.Equal("SELECT COUNT(*) FROM Starcounter.Linq.Tests.Person P WHERE ((P.Name = ?))",
```

Comment:
Please replace `COUNT(*)` with `COUNT(p)`. It does not really matter in this case, but in Starcounter `COUNT(*)` is more expensive than `COUNT(p)`.

Reply:
That's not necessarily the case: https://github.com/Starcounter/level1/issues/1670#issuecomment-326803000

Reply:
It is, because `COUNT(*)` is treated as a literal rather than a parameter. So, SQL queries generated by LINQ should be generated properly, using `COUNT(P)`.

Reply:
Replaced `COUNT(*)` with `COUNT(P)`